sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -142,7 +143,7 @@
       .add("is_available", ValueType.LONG)
       .add("is_realtime", ValueType.LONG)
       .add("is_overshadowed", ValueType.LONG)
-      .add("shardSpec", ValueType.STRING)
+      .add("shard_spec", ValueType.STRING)
```

Comment:
`shardSpec` was introduced in this change https://github.com/apache/druid/pull/9883 and changing it to `shard_spec` would be a breaking change. I don't think we should do this without an upgrade plan

Reply:
I don't think there is anything depending on shardSpec (similar to how the linked PR changed payload to shardSpec, metrics, dimensions). I can update the document but I don't think anything will break. I guess unless someone has a custom script reading this field which is probably unlikely and is why we should change this ASAP.

Reply:
The linked PR added those fields, so we didn't have to worry about backwards incompatibility.

@yuanlihan since you introduced this field, do you have any comments on changing the name of this field?

Reply:
The linked PR removed `payload` and added `shardSpec`, `metrics`, `dimensions`